plugin.video.amazon-test/resources/lib/startup.py

```diff
@@ -84,8 +83,8 @@ def EntryPoint():
         g.pv.Browse(None if 'path' not in args else args['path'])
     elif mode == 'PrimeVideo_Search':
         g.pv.Search()
-    elif None is not re.match(r'^[a-zA-Z]+$', mode):
+    elif re.match(r'^[a-zA-Z]+$', mode):
         cmd = mode + '()'
```

Comment:
This is still unsafe and exploitable. Are there unmapped method calls we need to make explicit, or can we get rid of this now?

Reply:
Oh just forget to add it. We can use this, to detect existing methods:
`elif mode in dir(g.amz) + globals().keys():`
That should be safer, then the old call.

Reply:
Done. I'm still not aware of any raw calls like that though, so for the moment I left the list empty.

Reply:
I had to changed this, cause of compatibility to Kodi 17. The `dir` call doesn't work as expected there. I hope, i didn't forget any method.